src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const personPartner = sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  const marriedInfo = isMarried
+    ? `${partnerName} is my ${personPartner}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age
```

Comment:
You have to explicitly state that it's a boolean, because it can just render 0:
```suggestion
      {Boolean(age)
```